p11-kit/lists.c

```diff
@@ -119,35 +120,49 @@ print_token_info (CK_FUNCTION_LIST_PTR module, CK_SLOT_ID slot_id)
 	}
 
 	value = p11_kit_space_strdup (info.label, sizeof (info.label));
-	printf ("    token: %s\n", value);
+	printf (isatty_out ?
+		"    \033[32;1mtoken:\033[0m %s\n" :
+		"    token: %s\n", value);
 	free (value);
 
 	value = p11_kit_space_strdup (info.manufacturerID, sizeof (info.manufacturerID));
-	printf ("        manufacturer: %s\n", value);
+	printf (isatty_out ?
+		"        \033[1mmanufacturer:\033[0m %s\n" :
```

Comment:
To avoid repetition, I would suggest creating a helper function something like:
```c
enum p11_print_level { ... };
static const char *level_to_sgr[] = { "\033[1m" };
#define SGR_RESET "\033[0m"

static void
p11_print_value (FILE *fp, const char *name, const char *value, size_t indent, enum p11_print_level level)
{
  // print indent

  if (isatty (fileno (fp)))
    // print SGR designation
  fprintf (fp, "%s:", name);
  if (isatty (fileno (fp)))
    // print SGR reset
  fprintf (fp, " %s\n", value);
}
```

Reply:
I have refactored the code and used helper functions, for now I am keeping the refactoring in a separate commit but I will squash them afterward.